tools/system_libs.py

```diff
@@ -1585,13 +1583,18 @@ def build_port(src_path, output_path, includes=[], flags=[], exclude_files=[], e
       commands.append([shared.PYTHON, shared.EMCC, '-c', src, '-O2', '-o', obj, '-w'] + include_commands + flags)
       objects.append(obj)
 
-    run_commands(commands)
+    Ports.run_commands(commands)
     create_lib(output_path, objects)
     return output_path
 
   @staticmethod
-  def run_commands(commands): # make easily available for port objects
-    run_commands(commands)
+  def run_commands(commands):
```

Comment:
How about renaming this? I think it's confusing to have two `run_commands` in difference namespaces that do different things.

Perhaps `run_with_cflags`?

Alternatively, to avoid the duplication of flags, we can remove the extra cflags from the other side (callers of toplevel `run_build_command`)?

Reply:
Or, how about removing the now-trivial `run_build_command` and make the callers call `run_process` directly? (do we need that extra logging? I think maybe not worth it.)

Reply:
`run_build_command` is only used as a helper for the parallel `run_commands` .. I'll move it into a local function to make that more clear.

Reply:
How about adding a comment that this is for Ports implementations and so adds cflags automatically? (with that it's clearer I think that this is a design choice, and not that at some point we added cflags in the implementation of the function)

Reply:
Done